styleguide/source/_patterns/01-molecules/06-components/11-banner-cta/banner-cta-no-stacking.twig

```diff
@@ -1,8 +1,8 @@
-<article class="banner-cta-no-stacking">
-    {% include 'atoms-h1' with { 'content': 'Lorem ipsum dolor sit amet', 'class': 'banner-cta-no-stacking_title' } %}
-    <div class="banner-cta-no-stacking_image" style="background-image: url(../../assets/images/fpo_3x2.png)"></div>
+<div class="banner-cta-no-stacking">
```

Comment:
Addressed the ipsumimage in [https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/f0440da79227576ef8cef72ff5dfbde7123240d1](this commit.)